extensions/answer_summarizers/models_test.py

```diff
@@ -0,0 +1,113 @@
+from core.tests import test_utils
+import feconf
+import schema_utils
+import utils
+
+
+class InteractionAnswerSummariesCalculationsTest(test_utils.GenericTestBase):
+    """Tests for ExpSummary aggregations."""
+
+    def test_answer_counts_calc_without_job(self):
+        """For multiple choice interaction, test if most common answers are
+        calculated correctly for interaction answer views."""
+
+        exp_id = '0'
+        exp_version = 1
+        state_name = 'Welcome!'
+        params = {}
+        DEFAULT_RULE_STR = exp_domain.DEFAULT_RULESPEC_STR
+        exp_services.load_demo(exp_id)
+
+        # Some answers
+        dummy_answers_list = [
+            {'answer_value': 'First choice', 'time_spent_in_sec': 4.,
+            'session_id': 'sid1'},
+            {'answer_value': 'Second choice', 'time_spent_in_sec': 5.,
+            'session_id': 'sid1'},
+            {'answer_value': 'Fourth choice', 'time_spent_in_sec': 2.5,
+            'session_id': 'sid1'},
+            {'answer_value': 'First choice', 'time_spent_in_sec': 10.,
+            'session_id': 'sid2'},
+            {'answer_value': 'First choice', 'time_spent_in_sec': 3.,
+            'session_id': 'sid2'},
+            {'answer_value': 'First choice', 'time_spent_in_sec': 1.,
+            'session_id': 'sid2'},
+            {'answer_value': 'Second choice', 'time_spent_in_sec': 20.,
+            'session_id': 'sid2'},
+            {'answer_value': 'First choice', 'time_spent_in_sec': 20.,
+            'session_id': 'sid3'}
+            ]
+
+        # Record answers
+        for answer in dummy_answers_list:
+            stats_services.record_answer(
+                exp_id, exp_version, state_name, 'test_handler', 
+                DEFAULT_RULE_STR, answer['session_id'], 
+                answer['time_spent_in_sec'], params, 
+                answer['answer_value'])
+
+        # Retrieve state answers from storage and get corresponding
+        # StateAnswers domain object.
+        state_answers = stats_services.get_state_answers(
+            exp_id, exp_version, state_name)
+        self.assertEquals(
+            state_answers.interaction_id, 'MultipleChoiceInput')
+
+        # Calculate answer counts. Input is dummy StateAnswersModel entity, 
+        # output is a list of dicts, each with keys 'answer' and 'frequency'.
+        calculation_instance = (
+            calculation_registry.Registry.get_calculation_by_id(
+                'AnswerCounts'))
+        actual_state_answers_calc_output = (
+            calculation_instance.calculate_from_state_answers_entity(
+                state_answers))
+
+        self.assertEquals(
+            actual_state_answers_calc_output.calculation_id, 'AnswerCounts')
+        actual_calc_output = actual_state_answers_calc_output.calculation_output
+
+        # Expected answer counts (result of calculation)
+        # TODO(msl): Maybe include answers that were never clicked and got 0 count
```

Comment:
I think you should remove 'Maybe' here, since TODOs should propose an actionable item rather than ask a question.